app/models/contact.rb

```diff
@@ -1,6 +1,10 @@
 class Contact < ActiveRecord::Base
+  EMBEDDED_CONTACT_REGEXP = /\[Contact\:([0-9]+)\]/
```

Comment:
ok, so why not put it in `GovspeakHelper::ContactsExtractor`? It's not used anywhere else as far as I could see?